upgrade/sql/8.0.0.sql

```diff
@@ -7,26 +7,18 @@ DROP TABLE IF EXISTS `PREFIX_referrer_shop`;
 
 /* Remove page Referrers */
 ## Remove Tabs
-DELETE FROM `PREFIX_tab` WHERE `class_name` = 'AdminReferrers';
-DELETE FROM `PREFIX_tab_lang` WHERE `id_tab` NOT IN (SELECT id_tab FROM `PREFIX_tab`);
+/* PHP:ps_remove_controller_tab('AdminThemesCatalog'); */;
+/* PHP:ps_remove_controller_tab('AdminModulesCatalog'); */;
+/* PHP:ps_remove_controller_tab('AdminAddonsCatalog'); */;
+/* PHP:ps_remove_controller_tab('AdminReferrers'); */;
 ## Remove Roles
-DELETE FROM `PREFIX_access` WHERE `id_tab` NOT IN (SELECT id_tab FROM `PREFIX_tab`);
```

Comment:
there is no ´id_tab´ colum in the ps_access table, I'm not sure what this PR was doing... is it a column that existed but was removed from schema?

Reply:
🤷‍♀️ 😅